### PR TITLE
implement access interface of ProjectApi

### PR DESCRIPTION
**Pull request by keylorsdu**

### CODE REVIEW

File: src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java [hunk 1]
```diff
@@ -119,8 +122,27 @@ public TagApi tag(String ref) throws RestApiException {
         JsonElement tags = gerritRestClient.getRequest(request);
         return tagInfoParser.parseTagInfos(tags);
     }
-
-
+    
+    /* get accesses of the project
```
Comment (uwolfer) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java]: You can remove this comment - it's also not there for other methods in this class. The interface should document it.

File: src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java [hunk 2]
```diff
@@ -119,8 +122,27 @@ public TagApi tag(String ref) throws RestApiException {
...
+     * @see com.google.gerrit.extensions.api.projects.ProjectApi.NotImplemented#access()
+     */
+    @Override
+    public ProjectAccessInfo access() throws RestApiException {
+    	String request = projectsUrl() + "/access";
+    	JsonElement result =  gerritRestClient.getRequest(request);
+    	return projectsParser.parseProjectAccessInfo(result);
+    }
+    
+    /* set access for the project
```
Comment (uwolfer) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java]: Same here.

File: src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java [hunk 3]
```diff
@@ -148,8 +170,8 @@ private String branchesUrl(ListRefsRequest<BranchInfo> lbr) {
         return url;
     }
 
-    private String tagsUrl(ListRefsRequest<TagInfo> lrr) {
         String query = "";
+        private String tagsUrl(ListRefsRequest<TagInfo> lrr) {
```
Comment (uwolfer) [inline src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java]: This looks wrong.

File: src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java (hunk 2 above)
Comment (keylorsdu) [reply src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java]: ok， I see,.

**Comment by keylorsdu** [comment]
I have write two simple test and remove the comment.

**Comment by uwolfer** [comment]
@keylorsdu: Thanks for the update. It looks like a merge went wrong. I suggest you to squash all commits into one commit and check the diff that there are no unnecessary changes. Then you can force-push to your branch in order to update this pull request.

**Pull request closed without merging**